pages/linux/gio-trash.md

```diff
@@ -0,0 +1,17 @@
+# gio trash
+
+> Move files to the trash bin.
+> Used by GNOME to handle trash.
+> More information: <https://gitlab.gnome.org/GNOME/glib>.
```

Comment:
```suggestion
> More information: <https://manned.org/gio.1>.
```

Reply:
The [official website for the documentation](https://docs.gtk.org/gio/) mentioned in the repo's README only documents the code, and not the CLI.

Reply:
Ok, I'll commit this suggestion.